roles/deploy/hooks/build-after.yml

```diff
@@ -21,3 +21,9 @@
   composer:
     no_scripts: yes
     working_dir: "{{ deploy_helper.new_release_path }}"
+
+- name: Gzip static assets
+  shell: "for i in `find . | grep -E \"\\.css$|\\.js$|\\.txt$|\\.html$|\\.xml$|\\.svg$|\\.ttf$|\\.otf$|\\.eot$|\\.woff$|\\.woff2$\"`; do gzip --best --keep --force \"$i\"; done"
```

Comment:
> Why `--force`?

If `.gz` files already exist for whatever reason, `gzip` without `--force` would prompt for confirmation. Since we using `--best`, our `.gz` files are... the best. Therefore, overriding existing `.gz` files seems harmless.

Actually, I am not familiar with shell commands. What command do you suggest?
Besides, we need to think about customization as well.